.github/ISSUE_TEMPLATE/launch-initial.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Launch
+about: Prepare for a site launch with a checklist of common steps.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
```

Comment:
Do we need to add anything to this ticket? Is this still a work in progress? Perhaps this is intended to replace `launch.md`?

Reply:
Ok, I have shuffled tasks around. I'm sure there is a lot missing and plenty of room for improvement, but it's a start!